### DISCUSSION TITLE
Aggiornamento

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#249] [Richieste di aiuto] [answered]
Salve, posso chiedere quali saranno le modifiche per il prossimo aggiornamento?

**Comment by trinko** [comment, marked as answer]
La gestione dello scrutinio finale: devo fare alcune modifiche per la conversione dei crediti per le quinte e, se riesco, comincio ad introdurre qualche modifica per renderlo un più generico e configurabile.
